osquery/tables/system/python_packages.cpp

```diff
@@ -122,11 +122,23 @@ void genWinPythonPackages(const std::string& keyGlob, QueryData& results) {
 #endif
 }
 
+
+
+
 QueryData genPythonPackages(QueryContext& context) {
   QueryData results;
-
-  for (const auto& key : kPythonPath) {
-    genSiteDirectories(key, results);
+  std::set<std::string> paths;
```

Comment:
This code can be refactored to not repeat the `genSiteDirectories()` loop. Make `paths` be `kPythonPath` if no constraints and then have the `genSiteDirectories()` only once.